Tests/LibGL/TestRender.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Debug.h>
+#include <AK/Format.h>
+#include <LibGL/GL/gl.h>
+#include <LibGL/GLContext.h>
+#include <LibGfx/BMPWriter.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/FontDatabase.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#define RENDER_WIDTH 16
+#define RENDER_HEIGHT 16
+
+TEST_CASE(simple_triangle)
+{
+    auto bitmap = Gfx::Bitmap::try_create(Gfx::BitmapFormat::BGRx8888, { RENDER_WIDTH, RENDER_HEIGHT }).release_value_but_fixme_should_propagate_errors();
```

Comment:
There's nowhere to propagate the error to, how about just
```suggestion
    auto bitmap = MUST(Gfx::Bitmap::try_create(Gfx::BitmapFormat::BGRx8888, { RENDER_WIDTH, RENDER_HEIGHT }));
```